youtube_dl/extractor/bbc.py

```diff
@@ -959,6 +959,16 @@ def _real_extract(self, url):
              r'videoId\s*:\s*["\'](%s)["\']' % self._ID_REGEX],
             webpage, 'vpid', default=None)
 
+        # bbc reel (e.g. https://www.bbc.com/reel/video/p07c6sb6/how-positive-thinking-is-harming-your-happiness)
+        initial_data = self._search_regex(
+            r'<script[^>]+id="initial-data"[^>]+data-json=\'(.+)\'>',
+            webpage, 'initial data', fatal=False, default=None)
```

Comment:
don't use both `fatal` and `default`.

Reply:
Sure thing. Looks like ajj8 didn't read the spec of `_search_regex`. Maybe I didn't either ...

Obviously not using `fatal=False` is what was intended: https://github.com/dirkf/youtube-dl-1/commit/b61ab55aacec21505c763b769d60a1c5ac8691a1.